kvstore/utils/globals.py

```diff
@@ -0,0 +1,8 @@
+from enums import Placement
+from dbr.logger import Logger
+
+# smart placement by default
+PLACEMENT = Placement.SMART
+LOGGER = Logger()
+CLIENT_LOCATION = "127.0.0.1"
+REGION_LATENCIES = {}  # TODO: fill in
```

Comment:
this we can just get from our config csv right?

Reply:
yep